src/test/java/uk/gov/companieshouse/util/TestUtil.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.companieshouse.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import uk.gov.companieshouse.model.Appeal;
+import uk.gov.companieshouse.model.OtherReason;
+import uk.gov.companieshouse.model.PenaltyIdentifier;
+import uk.gov.companieshouse.model.Reason;
+
+import java.io.File;
+
+public class TestUtil {
```

Comment:
Not big fan of this as it does everything and responsibilities are not split but you can leave it for now as is cause impact of this is low.

Reply:
Priority: do not change.